email_notification/templates/email/html/partials/base.jinja

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html lang="{{ language }}">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>{% block title %}Varaamo{% endblock %}</title>
+
+    <style>
+        @font-face {
+            font-family: "HelsinkiGrotesk";
+            src: url("https://makasiini.hel.ninja/delivery/HelsinkiGrotesk/565d73a693abe0776c801607ac28f0bf.woff") format("woff");
+        }
+
+        body {
+            background-color: #FFFFFF;
+            max-width: 960px;
+            margin: 0 auto;
+            font-family: HelsinkiGrotesk, serif;
+        }
+
+        p {
+            margin: 0;
+            font-size: 18px;
+        }
+
+        h2 {
+            font-size: 20px;
+            margin-top: 15px;
+            margin-bottom: 15px;
+        }
+
+        #header {
+            display: flex;
+            flex-direction: column;
+            gap: 50px;
+            padding: 30px 25px;
```

Comment:
`padding`: Window Outlook strikes again (`margin` is safer): https://www.caniemail.com/features/css-padding/